tests/integration/datadictwithobjid/submission/test_endpoints.py

```diff
@@ -355,7 +359,7 @@ def put_example_entities_together(client, headers):
     return client.put(path, headers=headers, data=json.dumps(data))
 
 
-def test_post_example_entities_together(client, pg_driver, cgci_blgsp, submitter):
+def do_test_post_example_entities_together(client, submitter):
```

Comment:
not importing it from `datadict` because the `datadictwithobjid` test data might be different